eval/src/vespa/eval/eval/node_types.cpp

```diff
@@ -307,7 +307,13 @@ struct TypeExporter : public NodeTraverser {
         : parent_type_map(parent_type_map_in),
           exported_type_map(exported_type_map_out),
           missing_cnt(0) {}
-    bool open(const Node &) override { return true; }
+    bool open(const Node &node) override {
+        if (auto lambda = as<TensorLambda>(node)) {
+            lambda->lambda().root().traverse(*this);
+            return false;
```

Comment:
we need to also bind our own type, unfortunately, removing return false is the simplest way, which is a bit confusing.